addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -1035,6 +1041,21 @@ def _recompute_move_entries_totals(env):
     )
 
 
+def fill_account_move_line_missing_fields(env):
+    openupgrade.logged_query(env.cr, """
```

Comment:
Have you tested if performing the UPDATE of both fields in one query joining with `LEFT JOIN` the 3 tables works faster?

Reply:
I am not sure if that will work, as one of the fields is not required. I am not sure what will happen in that case. On an update you cannot pass the value on LEFT JOIN / ON, you must use it on the WHERE, the query should be like
```sql
        UPDATE account_move_line aml
        SET 
        account_root_id = aa.root_id,
        tax_group_id = at.tax_group_id
        FROM account_tax at, account_account aa
        WHERE aa.id = aml.account_id AND at.id = aml.tax_line_id
```
but I cannot assure that it will work as expected

Reply:
no, that query is incorrect.